libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorReserve.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import { i18n } from '../../../core';
+import { IN_STORE_PICKUP_LABEL, IN_STORE_PICKUP } from '../../constants';
+import { StockInfo } from '../StockInfo';
+import { ChangeLocationButton } from '../ChangeLocationButton';
+import { itemRow, itemColumn } from './FulfillmentSelectorItem.style';
+import { useFulfillmentSelectorState } from './FulfillmentSelector.hooks';
+import { container } from './FulfillmentSelectorReserve.style';
+
+/**
+ * Renders the reservation item label.
+ * @returns {JSX}
+ */
+export function FulfillmentSelectorReserve() {
+  const { location, selectedLocation, handleChange } = useFulfillmentSelectorState();
+  const usedLocation = selectedLocation || location;
+
+  const handleChangeLocation = React.useCallback(() => {
+    handleChange(IN_STORE_PICKUP, true);
+  }, [handleChange]);
+
+  return (
+    <React.Fragment>
+      <div>
+        {i18n.text(IN_STORE_PICKUP_LABEL)}
+      </div>
+      {usedLocation && (
+        <div className={classNames(itemRow.toString(), container.toString())}>
+          <div className={itemColumn}>
+            <div>{usedLocation.name}</div>
+            <ChangeLocationButton onClick={handleChangeLocation} />
```

Comment:
The button is still clickable when radio is disabled
![image](https://user-images.githubusercontent.com/18292310/76841444-0edcd700-6839-11ea-9000-931c993214d3.png)

Instead it should be like this
![image](https://user-images.githubusercontent.com/18292310/76841530-33d14a00-6839-11ea-9e9c-e7094932afbd.png)

Reply:
Added, about the button, it should be clickable, I understood.